backend/models/contractorModel.js

```diff
@@ -44,6 +44,11 @@ const contractorSchema = new Schema({
     required: false,
     default: ''
   },
+  profile_name: {
```

Comment:
`profile_name` is already being represented by `name`. Or do you intend on using the fields in different ways?